packages/bundler/src/Config.ts

```diff
@@ -1,6 +1,6 @@
 import ow from 'ow'
 import fs from 'fs'
-
+import * as lodash from 'lodash'
```

Comment:
smalll optim: lodash is huge, and you use a single function. better `import from "lodash/merge"`, or something like that.